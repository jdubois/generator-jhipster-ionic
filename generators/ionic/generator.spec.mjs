import { beforeAll, describe, expect, it } from 'vitest';

import { defaultHelpers as helpers, result } from 'generator-jhipster/testing';

const SUB_GENERATOR = 'ionic';
const SUB_GENERATOR_NAMESPACE = `jhipster-ionic:${SUB_GENERATOR}`;

const expectedJwtFiles = [
  'src/app/services/auth/auth-jwt.service.ts',
  'src/app/services/auth/auth-jwt.service.spec.ts',
  'src/app/interceptors/auth-expired.interceptor.ts',
];

describe('SubGenerator ionic of ionic JHipster blueprint', () => {
  describe('with jwt authentication', () => {
    beforeAll(async function () {
      await helpers
        .run(SUB_GENERATOR_NAMESPACE)
        .withJHipsterConfig()
        .withOptions({
          blueprint: 'ionic',
          appDir: false,
          baseName: 'jhipster',
          ignoreNeedlesError: true,
          authenticationType: 'jwt',
        })
        .withJHipsterLookup()
        .withParentBlueprintLookup();
    });

    it('should succeed', () => {
      expect(result.getStateSnapshot()).toMatchSnapshot();
    });

    it('should generate app/services/auth/auth-jwt.service.ts', () => {
      result.assertFile(expectedJwtFiles);
    });
  });
  describe('with oauth2 authentication', () => {
    beforeAll(async function () {
      await helpers
        .run(SUB_GENERATOR_NAMESPACE)
        .withJHipsterConfig()
        .withOptions({
          blueprint: 'ionic',
          appDir: false,
          baseName: 'jhipster',
          authenticationType: 'oauth2',
          ignoreNeedlesError: true,
        })
        .withJHipsterLookup()
        .withParentBlueprintLookup();
    });

    it('should succeed', () => {
      expect(result.getStateSnapshot()).toMatchSnapshot();
    });

    it('should not generate jwt files', () => {
      result.assertNoFile(expectedJwtFiles);
    });
  });
});
