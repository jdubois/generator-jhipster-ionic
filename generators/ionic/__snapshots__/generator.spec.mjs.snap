// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SubGenerator ionic of ionic JHipster blueprint > with jwt authentication > should succeed 1`] = `
{
  ".browserslistrc": {
    "stateCleared": "modified",
  },
  ".editorconfig": {
    "stateCleared": "modified",
  },
  ".eslintrc.json": {
    "stateCleared": "modified",
  },
  ".gitattributes": {
    "stateCleared": "modified",
  },
  ".gitignore": {
    "stateCleared": "modified",
  },
  ".husky/pre-commit": {
    "stateCleared": "modified",
  },
  ".lintstagedrc.cjs": {
    "stateCleared": "modified",
  },
  ".prettierignore": {
    "stateCleared": "modified",
  },
  ".prettierrc.yml": {
    "stateCleared": "modified",
  },
  ".yo-rc.json": {
    "stateCleared": "modified",
  },
  "README.md": {
    "stateCleared": "modified",
  },
  "angular.json": {
    "stateCleared": "modified",
  },
  "capacitor.config.ts": {
    "stateCleared": "modified",
  },
  "cypress.config.ts": {
    "stateCleared": "modified",
  },
  "cypress/e2e/app.cy.ts": {
    "stateCleared": "modified",
  },
  "cypress/e2e/login.cy.ts": {
    "stateCleared": "modified",
  },
  "cypress/fixtures/integration-test.png": {
    "stateCleared": "modified",
  },
  "cypress/plugins/index.ts": {
    "stateCleared": "modified",
  },
  "cypress/support/commands.ts": {
    "stateCleared": "modified",
  },
  "cypress/support/config.ts": {
    "stateCleared": "modified",
  },
  "cypress/support/index.ts": {
    "stateCleared": "modified",
  },
  "cypress/support/pages/app.po.ts": {
    "stateCleared": "modified",
  },
  "cypress/support/pages/entity.po.ts": {
    "stateCleared": "modified",
  },
  "cypress/support/pages/login.po.ts": {
    "stateCleared": "modified",
  },
  "cypress/tsconfig.json": {
    "stateCleared": "modified",
  },
  "ionic.config.json": {
    "stateCleared": "modified",
  },
  "jest.config.js": {
    "stateCleared": "modified",
  },
  "package.json": {
    "stateCleared": "modified",
  },
  "src/app/app-routing.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/app.component.html": {
    "stateCleared": "modified",
  },
  "src/app/app.component.scss": {
    "stateCleared": "modified",
  },
  "src/app/app.component.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/app.component.ts": {
    "stateCleared": "modified",
  },
  "src/app/app.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/interceptors/auth-expired.interceptor.ts": {
    "stateCleared": "modified",
  },
  "src/app/interceptors/auth.interceptor.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/account/account.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/account/account.page.html": {
    "stateCleared": "modified",
  },
  "src/app/pages/account/account.page.scss": {
    "stateCleared": "modified",
  },
  "src/app/pages/account/account.page.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/account/account.page.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/entities/entities.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/entities/entities.page.html": {
    "stateCleared": "modified",
  },
  "src/app/pages/entities/entities.page.scss": {
    "stateCleared": "modified",
  },
  "src/app/pages/entities/entities.page.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/entities/entities.page.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/home/home.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/home/home.page.html": {
    "stateCleared": "modified",
  },
  "src/app/pages/home/home.page.scss": {
    "stateCleared": "modified",
  },
  "src/app/pages/home/home.page.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/home/home.page.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/login/login.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/login/login.page.html": {
    "stateCleared": "modified",
  },
  "src/app/pages/login/login.page.scss": {
    "stateCleared": "modified",
  },
  "src/app/pages/login/login.page.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/login/login.page.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/signup/signup.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/signup/signup.page.html": {
    "stateCleared": "modified",
  },
  "src/app/pages/signup/signup.page.scss": {
    "stateCleared": "modified",
  },
  "src/app/pages/signup/signup.page.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/signup/signup.page.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/tabs/tabs.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/tabs/tabs.page.html": {
    "stateCleared": "modified",
  },
  "src/app/pages/tabs/tabs.page.scss": {
    "stateCleared": "modified",
  },
  "src/app/pages/tabs/tabs.page.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/tabs/tabs.page.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/tabs/tabs.router.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/welcome/welcome.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/welcome/welcome.page.html": {
    "stateCleared": "modified",
  },
  "src/app/pages/welcome/welcome.page.scss": {
    "stateCleared": "modified",
  },
  "src/app/pages/welcome/welcome.page.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/welcome/welcome.page.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/api/api.service.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/api/api.service.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/auth/account.service.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/auth/account.service.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/auth/auth-jwt.service.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/auth/auth-jwt.service.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/auth/user-route-access.service.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/auth/user-route-access.service.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/login/login.service.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/login/login.service.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/user/user.model.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/user/user.service.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/user/user.service.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/utils/data-util.service.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/utils/data-util.service.ts": {
    "stateCleared": "modified",
  },
  "src/app/shared/index.ts": {
    "stateCleared": "modified",
  },
  "src/app/shared/util/request-util.ts": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/ar.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/bs.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/by.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/da.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/de.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/el.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/en.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/es-eu.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/es.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/fil.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/fr.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/he.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/it.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/ja.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/nb.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/nl.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/pl.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/pt-PT.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/pt-br.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/ru.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/sk.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/sl.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/sn.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/sv.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/th.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/tr.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/ua.json": {
    "stateCleared": "modified",
  },
  "src/assets/icon/favicon.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/appicon.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/hipster.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/hipster2x.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/ionic-logo.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster-logo.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_0.svg": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_0_head-192.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_0_head-256.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_0_head-384.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_0_head-512.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_1.svg": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_1_head-192.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_1_head-256.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_1_head-384.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_1_head-512.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_2.svg": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_2_head-192.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_2_head-256.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_2_head-384.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_2_head-512.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_3.svg": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_3_head-192.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_3_head-256.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_3_head-384.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_3_head-512.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/logo.png": {
    "stateCleared": "modified",
  },
  "src/assets/shapes.svg": {
    "stateCleared": "modified",
  },
  "src/environments/environment.prod.ts": {
    "stateCleared": "modified",
  },
  "src/environments/environment.ts": {
    "stateCleared": "modified",
  },
  "src/global.scss": {
    "stateCleared": "modified",
  },
  "src/index.html": {
    "stateCleared": "modified",
  },
  "src/main.ts": {
    "stateCleared": "modified",
  },
  "src/model/account.model.ts": {
    "stateCleared": "modified",
  },
  "src/model/base-entity.ts": {
    "stateCleared": "modified",
  },
  "src/polyfills.ts": {
    "stateCleared": "modified",
  },
  "src/theme/variables.scss": {
    "stateCleared": "modified",
  },
  "src/zone-flags.ts": {
    "stateCleared": "modified",
  },
  "tsconfig.app.json": {
    "stateCleared": "modified",
  },
  "tsconfig.json": {
    "stateCleared": "modified",
  },
  "tsconfig.spec.json": {
    "stateCleared": "modified",
  },
  "tslint.json": {
    "stateCleared": "modified",
  },
}
`;

exports[`SubGenerator ionic of ionic JHipster blueprint > with oauth2 authentication > should succeed 1`] = `
{
  ".browserslistrc": {
    "stateCleared": "modified",
  },
  ".editorconfig": {
    "stateCleared": "modified",
  },
  ".eslintrc.json": {
    "stateCleared": "modified",
  },
  ".gitattributes": {
    "stateCleared": "modified",
  },
  ".gitignore": {
    "stateCleared": "modified",
  },
  ".husky/pre-commit": {
    "stateCleared": "modified",
  },
  ".lintstagedrc.cjs": {
    "stateCleared": "modified",
  },
  ".prettierignore": {
    "stateCleared": "modified",
  },
  ".prettierrc.yml": {
    "stateCleared": "modified",
  },
  ".yo-rc.json": {
    "stateCleared": "modified",
  },
  "README.md": {
    "stateCleared": "modified",
  },
  "angular.json": {
    "stateCleared": "modified",
  },
  "capacitor.config.ts": {
    "stateCleared": "modified",
  },
  "cypress.config.ts": {
    "stateCleared": "modified",
  },
  "cypress/e2e/app.cy.ts": {
    "stateCleared": "modified",
  },
  "cypress/e2e/login.cy.ts": {
    "stateCleared": "modified",
  },
  "cypress/fixtures/integration-test.png": {
    "stateCleared": "modified",
  },
  "cypress/plugins/index.ts": {
    "stateCleared": "modified",
  },
  "cypress/support/commands.ts": {
    "stateCleared": "modified",
  },
  "cypress/support/config.ts": {
    "stateCleared": "modified",
  },
  "cypress/support/index.ts": {
    "stateCleared": "modified",
  },
  "cypress/support/oauth2.ts": {
    "stateCleared": "modified",
  },
  "cypress/support/pages/app.po.ts": {
    "stateCleared": "modified",
  },
  "cypress/support/pages/entity.po.ts": {
    "stateCleared": "modified",
  },
  "cypress/support/pages/login.po.ts": {
    "stateCleared": "modified",
  },
  "cypress/tsconfig.json": {
    "stateCleared": "modified",
  },
  "ionic.config.json": {
    "stateCleared": "modified",
  },
  "jest.config.js": {
    "stateCleared": "modified",
  },
  "package.json": {
    "stateCleared": "modified",
  },
  "src/app/app-routing.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/app.component.html": {
    "stateCleared": "modified",
  },
  "src/app/app.component.scss": {
    "stateCleared": "modified",
  },
  "src/app/app.component.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/app.component.ts": {
    "stateCleared": "modified",
  },
  "src/app/app.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/auth/auth-callback/auth-callback.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/auth/auth-callback/auth-callback.page.ts": {
    "stateCleared": "modified",
  },
  "src/app/auth/auth-config.service.ts": {
    "stateCleared": "modified",
  },
  "src/app/auth/auth-guard.service.ts": {
    "stateCleared": "modified",
  },
  "src/app/auth/auth-http.service.ts": {
    "stateCleared": "modified",
  },
  "src/app/auth/auth.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/auth/cap-http.service.ts": {
    "stateCleared": "modified",
  },
  "src/app/auth/end-session/end-session.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/auth/end-session/end-session.page.ts": {
    "stateCleared": "modified",
  },
  "src/app/auth/factories/auth.factory.ts": {
    "stateCleared": "modified",
  },
  "src/app/auth/factories/browser.factory.ts": {
    "stateCleared": "modified",
  },
  "src/app/auth/factories/http.factory.ts": {
    "stateCleared": "modified",
  },
  "src/app/auth/factories/index.ts": {
    "stateCleared": "modified",
  },
  "src/app/auth/factories/storage.factory.ts": {
    "stateCleared": "modified",
  },
  "src/app/auth/ng-http.service.ts": {
    "stateCleared": "modified",
  },
  "src/app/auth/user-info.model.ts": {
    "stateCleared": "modified",
  },
  "src/app/interceptors/auth.interceptor.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/account/account.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/account/account.page.html": {
    "stateCleared": "modified",
  },
  "src/app/pages/account/account.page.scss": {
    "stateCleared": "modified",
  },
  "src/app/pages/account/account.page.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/account/account.page.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/entities/entities.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/entities/entities.page.html": {
    "stateCleared": "modified",
  },
  "src/app/pages/entities/entities.page.scss": {
    "stateCleared": "modified",
  },
  "src/app/pages/entities/entities.page.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/entities/entities.page.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/home/home.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/home/home.page.html": {
    "stateCleared": "modified",
  },
  "src/app/pages/home/home.page.scss": {
    "stateCleared": "modified",
  },
  "src/app/pages/home/home.page.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/home/home.page.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/login/login.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/login/login.page.html": {
    "stateCleared": "modified",
  },
  "src/app/pages/login/login.page.scss": {
    "stateCleared": "modified",
  },
  "src/app/pages/login/login.page.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/login/login.page.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/tabs/tabs.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/tabs/tabs.page.html": {
    "stateCleared": "modified",
  },
  "src/app/pages/tabs/tabs.page.scss": {
    "stateCleared": "modified",
  },
  "src/app/pages/tabs/tabs.page.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/tabs/tabs.page.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/tabs/tabs.router.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/welcome/welcome.module.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/welcome/welcome.page.html": {
    "stateCleared": "modified",
  },
  "src/app/pages/welcome/welcome.page.scss": {
    "stateCleared": "modified",
  },
  "src/app/pages/welcome/welcome.page.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/pages/welcome/welcome.page.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/api/api.service.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/api/api.service.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/auth/account.service.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/auth/account.service.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/auth/user-route-access.service.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/auth/user-route-access.service.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/login/login.service.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/login/login.service.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/user/user.model.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/user/user.service.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/user/user.service.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/utils/data-util.service.spec.ts": {
    "stateCleared": "modified",
  },
  "src/app/services/utils/data-util.service.ts": {
    "stateCleared": "modified",
  },
  "src/app/shared/index.ts": {
    "stateCleared": "modified",
  },
  "src/app/shared/util/request-util.ts": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/ar.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/bs.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/by.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/da.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/de.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/el.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/en.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/es-eu.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/es.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/fil.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/fr.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/he.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/it.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/ja.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/nb.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/nl.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/pl.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/pt-PT.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/pt-br.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/ru.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/sk.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/sl.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/sn.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/sv.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/th.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/tr.json": {
    "stateCleared": "modified",
  },
  "src/assets/i18n/ua.json": {
    "stateCleared": "modified",
  },
  "src/assets/icon/favicon.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/appicon.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/hipster.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/hipster2x.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/ionic-logo.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster-logo.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_0.svg": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_0_head-192.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_0_head-256.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_0_head-384.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_0_head-512.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_1.svg": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_1_head-192.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_1_head-256.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_1_head-384.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_1_head-512.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_2.svg": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_2_head-192.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_2_head-256.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_2_head-384.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_2_head-512.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_3.svg": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_3_head-192.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_3_head-256.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_3_head-384.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/jhipster_family_member_3_head-512.png": {
    "stateCleared": "modified",
  },
  "src/assets/img/logo.png": {
    "stateCleared": "modified",
  },
  "src/assets/shapes.svg": {
    "stateCleared": "modified",
  },
  "src/environments/environment.prod.ts": {
    "stateCleared": "modified",
  },
  "src/environments/environment.ts": {
    "stateCleared": "modified",
  },
  "src/global.scss": {
    "stateCleared": "modified",
  },
  "src/index.html": {
    "stateCleared": "modified",
  },
  "src/main.ts": {
    "stateCleared": "modified",
  },
  "src/model/account.model.ts": {
    "stateCleared": "modified",
  },
  "src/model/base-entity.ts": {
    "stateCleared": "modified",
  },
  "src/polyfills.ts": {
    "stateCleared": "modified",
  },
  "src/theme/variables.scss": {
    "stateCleared": "modified",
  },
  "src/zone-flags.ts": {
    "stateCleared": "modified",
  },
  "tsconfig.app.json": {
    "stateCleared": "modified",
  },
  "tsconfig.json": {
    "stateCleared": "modified",
  },
  "tsconfig.spec.json": {
    "stateCleared": "modified",
  },
  "tslint.json": {
    "stateCleared": "modified",
  },
}
`;
